<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NOCalFactorsForm</class>
 <widget class="QWidget" name="NOCalFactorsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>5</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select High or Low NO Cal Factors</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lowButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Low NO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>High NO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Slope and offset for NO &lt;= 100 ppb</string>
   </property>
  </widget>
  <widget class="QLabel" name="highLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Slope and offset for NO &gt;= 101 ppb</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
